app/src/main/java/com/cornellappdev/uplift/ui/screens/SignInPromptScreen.kt

```diff
@@ -102,20 +107,22 @@ fun SignInPromptScreen() {
             UpliftUsesCardList()
 
             Spacer(modifier = Modifier.height(125.dp))
-            SigninButton({})
-
-
+            SignInButton(
+                onClick = { navController.navigate(route = "profileCreation") },
+                nextOnClick = { navController.navigate(route = "home") })
         }
     }
 }
 
 @Composable
-private fun SigninButton(onClick: () -> Unit) {
+private fun SignInButton(onClick: () -> Unit, nextOnClick: () -> Unit) {
     Button(
```

Comment:
I think the login button is a little wider with larger border radius on the figma